### PR TITLE
Test exported genesis

### PR DESCRIPTION
**Pull request by ASoTNetworks**

### CODE REVIEW

**Comment by uditvira** [comment]
Let's add this kind of info to readme:
- This repository automatically tests x, y, z
- Using GH actions we test upgrades with fresh state – badge
- On a remote VM we test upgrades with actual exported genesis files from Cosmos Hub mainnet – links to the log

File: .github/workflows/test-exported-genesis.yml [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+          node_key_file=examples/validator-keys/validator-40/node_key.json \
+          genesis_file=$latest_genesis"
+      - name: Check cosmovisor service
+        run: systemctl status cosmovisor
+      - name: Check blocks are being produced
+        run: tests/test_block_production.sh 127.0.0.1 26657 10
+      - name: Get RAM usage while gaiad is running
+        run: free -m
+
+  # test-upgrade:
```
Comment (makeworld-the-better-one) [inline .github/workflows/test-exported-genesis.yml:114]: Remove unused workflow

**Review by makeworld-the-better-one — APPROVED** [review]
shellcheck has some comments about the shell scripts in general, I would run them through that and go through the output if you haven't already.

File: tests/test_gaia_response.sh [hunk 2]
```diff
@@ -6,7 +6,7 @@ gaia_port=$2
 
 # Waiting until gaiad responds
 attempt_counter=0
-max_attempts=60
+max_attempts=1200
```
[the remark this answers is not in the document]
Comment (ASoTNetworks) [reply tests/test_gaia_response.sh:9]: same as above.

**Pull request merged**